extensions/applicationinsights-react-native/Tests/UnitTests.html

```diff
@@ -8,12 +8,12 @@
     <link rel="stylesheet" href="https://code.jquery.com/qunit/qunit-1.23.1.css">
     <script src="http://sinonjs.org/releases/sinon-2.3.8.js"></script>
     <script src="http://cdnjs.cloudflare.com/ajax/libs/require.js/2.2.0/require.js"></script>
-    <script src="../../../../common/Tests/Selenium/ModuleLoader.js"></script>
-    <script src="../../../../common/Tests/Selenium/SimpleSyncPromise.js"></script>
+    <script src="../../../common/Tests/Selenium/ModuleLoader.js"></script>
```

Comment:
Required as re-organizing the test files to be the same as the main sku's and extensions